app/code/Magento/Quote/Model/GuestCart/GuestCartManagement.php

```diff
@@ -8,7 +8,6 @@
 
 use Magento\Quote\Api\GuestCartManagementInterface;
 use Magento\Quote\Api\CartManagementInterface;
-use Magento\Quote\Model\QuoteIdMask;
```

Comment:
We can't remove this line because `Magento\Quote\Model\QuoteIdMask` used in `assignCustomer` method without the fully qualified name